### DISCUSSION TITLE
About WalletConnect Cloud project ID

### DISCUSSION THREAD

**Discussion by KBKUN024** [WalletConnect/walletconnect-monorepo#4083] [Developer Support]
I have created a project in WalletConnect Cloud and can see my project ID. When I use Web3Modal in a local Nuxt3 project, I can connect to the web version of MetaMask using my project ID. However, when I run this project on mobile, I cannot connect to MetaMask or imToken on my phone. Why is this?

I found another project that can connect to the wallet on the phone. When I pasted the project ID of this project into my Nuxt3 project, something magical happened - my Nuxt3 project can also connect to the mobile wallet. So my questions are:

1. Is it necessary to use domain verification for the project ID? I don't have domain verification yet.
2. Is there a tool that can check whether the project ID has been authenticated?
3. After I did doman verification,can I use the project ID in my local project,right?

There are some codes of my nuxt3 project:

### app.vue:
```vue
<script setup lang="ts">
import { createWeb3Modal, defaultConfig } from '@web3modal/ethers5/vue'

// 1. Get projectId
const projectId = 'cd4b190e1ff14f162550974018591ad0' // this is my project ID,when I use it,I can't connect to my mobile wallet
// const projectId = '80c55c1a9efb7ba1d32582051978a81d' // this one can connect to mobile wallet,but it's not mine,I found it on github

// 2. Set chains
const chains = [
  {
    chainId: 1,
    name: 'Ethereum',
    currency: 'ETH',
    explorerUrl: 'https://etherscan.io',
    rpcUrl: 'https://cloudflare-eth.com'
  },
  {
    chainId: 42161,
    name: 'Arbitrum',
    currency: 'ETH',
    explorerUrl: 'https://arbiscan.io',
    rpcUrl: 'https://arb1.arbitrum.io/rpc'
  }
]

// 3. Create modal
const metadata = {
  name: 'Web3Modal',
  description: 'Web3Modal Laboratory',
  url: 'https://web3modal.com',
  icons: ['https://avatars.githubusercontent.com/u/37784886']
}

createWeb3Modal({
  ethersConfig: defaultConfig({
    metadata,
    defaultChainId: 1,
    enableEIP6963: true,
    enableInjected: true,
    enableCoinbase: true,
    rpcUrl: 'https://cloudflare-eth.com' // used for the Coinbase SDK
  }),
  chains,
  projectId
})
</script>
```
### index.vue
```vue
<script setup lang="ts">
import { useWeb3Modal, useDisconnect, useWeb3ModalAccount } from '@web3modal/ethers5/vue'
const modal = useWeb3Modal()
const { disconnect } = useDisconnect()
const { address } = useWeb3ModalAccount()
console.log('dis:', disconnect);
console.log('address:', address.value);

</script>

<template>
  <div class=" flex flex-col">
    <w3m-button />
    {{ address }}
    <button @click="modal.open()">Open Connect Modal</button>
    <button @click="modal.open({ view: 'Networks' })">Open Network Modal</button>
    <button @click="disconnect">disconnect</button>
  </div>
</template>

```

Someone who can help me?I will so approciate it!

**Comment by KBKUN024** [comment]
@glitch-txs Can you help me?

**Reply by glitch-txs** [reply]
How did you test it on mobile on prod or local IP? Did you try setting up a new project with a new project ID?

**Comment by KBKUN024** [comment]
> How did you test it on mobile on prod or local IP? Did you try setting up a new project with a new project ID?
question1: I run my nuxt2 project,configuare my nuxt.config.js file so I can access my project on mobile
question2:No, I just created one project in walletconnect cloud, I will create a new one to test it can work or not
